services/frontend/src/components/NewsData.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="news-table-box">
+    <v-btn @click="createNew()"> <v-icon>mdi-plus</v-icon> Dodaj </v-btn>
+    <v-simple-table :loading="load">
+      <thead>
+        <tr>
+          <th>Tytuł</th>
+          <th>Data dodania</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-bind:key="singleNews.id" v-for="singleNews in news">
+          <td>{{ singleNews.title }}</td>
+          <td>{{ formatDate(singleNews.creation_date) }}</td>
+          <td>
+            <v-btn @click="goToNews(singleNews.id)">
+              <v-icon>mdi-note-search</v-icon> Edytuj
+            </v-btn>
+            <v-btn @click="removeNews(singleNews.id)">
+              <v-icon>mdi-delete</v-icon> Usuń
+            </v-btn>
+          </td>
+        </tr>
+      </tbody>
+    </v-simple-table>
+  </div>
+</template>
+
+<script>
+import { formatDate } from "../utils";
+import { api } from "@/api";
+
+export default {
+  name: "newsResultTable",
+  props: ["loading"],
```

Comment:
this is not used in the view, and you are getting data in this component , so this is redundant, in my opinion